### PR TITLE
Fixes #28587 - Test core with Ruby 2.7

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
As Rails 6 is in, I'd like to test with Ruby 2.7 now :)

**Comment by mmoll** [comment]
sure, rebased :)

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-plugins-pull-request.yaml [hunk 1]
```diff
@@ -37,7 +38,7 @@
             - fast
     execution-strategy:
       combination-filter: >
-        ((database == 'postgresql') && ((ruby == '2.5') || (ruby == '2.6'))) || ((database == 'sqlite3') && (ruby == '2.5'))
+        ((database == 'postgresql') && ((ruby == '2.5') || (ruby == '2.6') || (ruby == '2.7')) || ((database == 'sqlite3') && (ruby == '2.5'))
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-plugins-pull-request.yaml:41]: In 03c7f4227ebc4509149122160d9fccdb71ab790a we dropped sqlite3 testing for core - should we also drop it for plugins?

If not, we can still simplify this to:

```suggestion
        (database == 'postgresql') || ((database == 'sqlite3') && (ruby == '2.5'))
```

cc @tbrisker

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_plugin_matrix.yaml [hunk 2]
```diff
@@ -55,7 +56,7 @@
             - fast
     execution-strategy:
       combination-filter: >
-        ((database == 'postgresql') && ((ruby == '2.5') || (ruby == '2.6'))) || ((database == 'sqlite3') && (ruby == '2.5'))
+        ((database == 'postgresql') && ((ruby == '2.5') || (ruby == '2.6') || (ruby == '2.7'))) || ((database == 'sqlite3') && (ruby == '2.5'))
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_plugin_matrix.yaml:59]:
```suggestion
        (database == 'postgresql') || ((database == 'sqlite3') && (ruby == '2.5'))
```

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-plugins-pull-request.yaml (hunk 1 above)
Comment (mmoll) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-plugins-pull-request.yaml:41]: Sure, but please in a separate PR.

Comment (ekohl) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-plugins-pull-request.yaml:41]: Note my suggestion still keeps sqlite, just drops a redundant comparison for PG

Comment (mmoll) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-plugins-pull-request.yaml:41]: Ah, yes. My answer was to "should we also drop it for plugins?".

Comment (mmoll) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-plugins-pull-request.yaml]: > Note my suggestion still keeps sqlite, just drops a redundant comparison for PG

there we go...

**Pull request merged**